src/store/modules/accounts/ledger.js

```diff
@@ -70,24 +70,23 @@ export default {
     async signTransaction({ rootGetters: { account, 'sdkPlugin/sdk': sdk }, dispatch }, { txBase64 }) {
       await dispatch('ensureCurrentAccountAvailable');
 
-      const txObject = TxBuilder.unpackTx(txBase64).tx;
-      const binaryTx = TxBuilder.buildTx(
+      const txObject = unpackTx(txBase64).tx;
+      const encodedTx = buildTx(
         txObject,
-        SCHEMA.OBJECT_ID_TX_TYPE[txObject.tag],
+        Tag[txObject.tag],
         { vsn: txObject.VSN },
       ).rlpEncoded;
```

Comment:
```suggestion
      );
```
and wrap it with `decode()`